lib/hyperswitch.js

```diff
@@ -241,39 +218,39 @@ HyperSwitch.prototype._checkInternalApiRequest = function(req) {
     }
 };
 
-HyperSwitch.prototype.request = function(req, options) {
-    if (req.method) {
-        req.method = req.method.toLowerCase();
-    }
-    return this._request(req, options);
-};
+HyperSwitch.prototype._createFilteredHandler = function(filters, handler, specInfo) {
```

Comment:
Minor / matter of taste: My instinct would be to swap filters & handler here, but I don't have a good reason for it.